src/messages/javascript.js

```diff
@@ -240,6 +240,17 @@ export const KNOWN_LIBRARY = {
   ],
 };
 
+export const UNSAFE_DYNAMIC_VARIABLE_ASSIGNMENT = {
+  code: 'UNSAFE_VAR_ASSIGNMENT',
```

Comment:
There is currently a test that requires it and since it won't matter I just added it for now until we get rid of codes in favor of rule-ids.